features/kinderen.feature

```diff
@@ -1,71 +1,75 @@
-    En het kind van de ingeschreven persoon heeft volgens categorie 09/59 naam Chantal Groenen, geboren in Istanbul en geboortedatum 2 januari 2002
-    Als de kinderen worden geraadpleegd van de ingeschreven persoon met burgerservicenummer 999999291
-    Dan wordt het kind gevonden met naam.voornamen=Chantal
-    En heeft dit kind burgerservicenummer=null
-    En heeft dit kind naam.voorvoegsel=null
-    En heeft dit kind naam.geslachtsnaam=Groenen
-    En heeft dit kind naam.adellijkeTitel_predikaat.omschrijvingAdellijkeTitel_predikaat=null
-    En heeft dit kind geboorte.datum.datum=2002-01-02
-    En heeft dit kind geboorte.datum.dag=02
-    En heeft dit kind geboorte.datum.maand=01
-    En heeft dit kind geboorte.plaats=Istanbul
-    En heeft dit kind geboorte.land.landnaam=Turkije
-    En heeft dit gevonden kind een lege link ingeschrevenpersonen
+  Rule: Een kind wordt alleen teruggegeven als minimaal één gegeven in de identificatienummers (groep 01), naam (groep 02) of geboorte (groep 03) van het kind een waarde heeft.
+    # wanneer in een categorie kind alleen gegevens zijn opgenomen in groep 81 of 82, 85 en 86, wordt dit kind niet opgenomen in het antwoord
+    # wanneer een gegeven een onbekendwaarde heeft, zoals . (punt) bij geslachtsnaam en 00000000 bij geboortedatum, wordt die in deze regel beschouwd als leeg (geen waarde).
 
-  Scenario: de ingeschreven persoon twee kinderen
-    Gegeven de te raadplegen persoon heeft meerdere (twee) kinderen (Lisa en Daan)
-    Als de kinderen worden geraadpleegd van de ingeschreven persoon met burgerservicenummer 999999047
-    Dan is het aantal gevonden kinderen 2
-    En wordt het kind gevonden met naam.voornamen=Lisa
-    En wordt het kind gevonden met naam.voornamen=Daan
+    Scenario: kind volledig onbekend
+      Gegeven de persoon met burgerservicenummer 555550001 heeft de volgende kinderen in de registratie
+        | Categorie | Voornamen (02.10)  | Voorvoegsel (02.30) | Geslachtsnaam (02.40) | Geboortedatum (03.10) |
+        | 9         |                    |                     | .                     |                       |
+      En er zijn geen andere gegevens geregistreerd over de identificatienummers (groep 01), naam (groep 02) of geboorte (groep 03) in categorie 9
+      Als de ingeschreven persoon met burgerservicenummer 555550001 wordt geraadpleegd met fields=kinderen
+      Dan bevat het antwoord GEEN kinderen
 
-  Scenario: de ingeschreven persoon heeft drie kinderen
-    Gegeven de te raadplegen persoon heeft drie kinderen (Jip, Chantal en Jesse)
-    Als de kinderen worden geraadpleegd van de ingeschreven persoon met burgerservicenummer 999999291
-    Dan is het aantal gevonden kinderen 3
-    En wordt het kind gevonden met naam.voornamen=Jip
-    En wordt het kind gevonden met naam.voornamen=Chantal
-    En wordt het kind gevonden met naam.voornamen=Jesse
+    Scenario: ontkenning ouderschap
```

Comment:
Ik vraag mij af of dit functioneel correct is. Een consumer kan hierdoor niet zien of een persoon geen kind heeft of dat een persoon om een andere reden geen kind heeft.

Reply:
@CathyDingemanse  zoekt dat uit, zie https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/pull/848#discussion_r783949385